apidef/oas/server.go

```diff
@@ -36,6 +47,13 @@ func (s *Server) ExtractTo(api *apidef.APIDefinition) {
 	} else {
 		api.UseKeylessAccess = true
 	}
+
+	if s.ClientCertificates != nil {
+		s.ClientCertificates.ExtractTo(api)
+	} else {
```

Comment:
`else` is not necessary here because we don't omit in old API definition. Why did we set `api.UseKeylessAccess = true` above is because `api.UseKeylessAccess = !authentication.Enabled`

Reply:
just this is enough.

```
if s.ClientCertificates != nil {
		s.ClientCertificates.ExtractTo(api)
	}
```